components/Wishlists.php

```diff
@@ -87,7 +87,11 @@ public function onRun()
         /** @var Collection<Wishlist>|Wishlist[] items */
         /** @var Wishlist currentItem */
         $this->items       = $this->getWishlists();
-        $this->currentItem = $this->items->first();
+        $this->currentItem = $this->items->find($this->decode($this->param('id')));
+
+        if (! $this->currentItem) {
+            $this->currentItem = $this->items->first();
+        }
```

Comment:
```suggestion
        $this->currentItem = $this->items->find($this->decode($this->param('id'))) ?: $this->items->first();
```

Reply:
Thank's @mjauvin  to remember me the shorthand ternary operator! 👍